daprdocs/content/en/reference/components-reference/supported-bindings/kafka.md

```diff
@@ -54,6 +56,7 @@ spec:
 | saslPassword | N | Input/Output | The SASL password used for authentication. Can be `secretKeyRef` to use a [secret reference]({{< ref component-secrets.md >}}). Only required if `authRequired` is set to `"true"`. | `""`, `"KeFg23!"` |
 | initialOffset   | N | The initial offset to use if no offset was previously committed. Should be "newest" or "oldest". Defaults to "newest". | `"oldest"` |
 | maxMessageBytes | N | Input/Output | The maximum size in bytes allowed for a single Kafka message. Defaults to 1024. | `2048` |
+| version | N | Kafka cluster version. Defaults to 1.0.0.0 | `1.0.0`
```

Comment:
Looks like you have an extra 0 here. Should this be "Defaults to 1.0.0"

Reply:
I do not understand. Looks like there are two default values here. 1.0.0.0 and 1.0.0. Which is correct?

Reply:
1.0.0

Reply:
So can you change the 1.0.0.0 to 1.0.0 in the "Binding support" column?

Reply:
OK